mvp-spec/dsp-api.md

```diff
@@ -618,7 +618,7 @@ The DSP must build a valid Audit Log. For this purpose, it must:
 1. Take the Seed and the Transmission Parents from the Transmission Request that
 it received for this Addressable content.
 2. Consider the Transmission Parent to Transmission Result
-3. Append it own Transmission Result ot the existing list.
+3. Append it own Transmission Result to the existing list.
```

Comment:
If fixing this typo, you might use the opportunity to fix "it" to "its" 😉

Reply:
> If fixing this typo, you might use the opportunity to fix "it" to "its" wink

Done. Sorry, my eyes just clung to "ot" - I should have read more carefully :)

Reply:
😉